scheduler-portal/src/main/webapp/scheduler-portal.conf

```diff
@@ -0,0 +1,6 @@
+execution-list-extra-columns: [{ \
+	"title": "start at", \
+	"information": { \
+		"type": "generic-information", \
+		"key": "START_AT"}, \
+	"hidden-default": false }]
```

Comment:
Personally, I think that "hide" or "visible" would be much simpler. If required, the behavior could be described in the documentation.